schema/snapcraft.yaml

```diff
@@ -190,6 +190,10 @@ properties:
                 - type: string
                   minLength: 1
                 - type: number
+          no-wrapper:
```

Comment:
Agreed, this should be an array of unique elements like `build-attributes` in parts.